netaddr_test.go

```diff
@@ -760,6 +760,41 @@ func TestLessCompare(t *testing.T) {
 	}
 }
 
+func TestIPStringExpanded(t *testing.T) {
```

Comment:
Benchmark? Do we bench String currently? No need to add a bench if no, but if we test String's perf we should do the same with StringExpanded.

Reply:
Added.